src/foam/nanos/u2/navigation/SignIn.js

```diff
@@ -182,6 +185,60 @@ foam.CLASS({
           type: type
         }));
       }
+    },
+    {
+      name: 'signInWithOIDC',
+      code: async function(provider) {
+        // TODO: Validate nonce
+        var nonce = crypto.randomUUID();
+
+        var reqParams = {
+          response_type: 'code',
+          client_id: provider.clientId,
+          scope: 'openid email',
```

Comment:
not sure if its necessary for this pr, but would be good if this was configurable somehow ... e.g for intuit openid we also used scope for fullname, phone number)

Reply:
or maybe it is not necessary at all for just sign in